driver/src/main/scala/com/stratio/sparkta/driver/repository/ClusterConfigRepositoryComponent.scala

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (C) 2016 Stratio (http://stratio.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.stratio.sparkta.driver.repository
+
+import com.typesafe.config.Config
+
+import com.stratio.common.utils.components.config.impl.TypesafeConfigComponent
+import com.stratio.common.utils.components.logger.impl.Slf4jLoggerComponent
+import com.stratio.common.utils.components.repository.impl.ZookeeperRepositoryComponent
+
+class ClusterConfigRepositoryComponent(conf: Config) extends ZookeeperRepositoryComponent with
+TypesafeConfigComponent
+with Slf4jLoggerComponent {
+
+  override val config = new TypesafeConfig(Option(conf))
+
+  def upsert(entity: String, id: String, element: Array[Byte]): Unit = {
```

Comment:
I create this function in the common utils. The next release will contain this feature.
Sorry, we forget this necessary function.